src/restruct_poc/solver/populate_sparse_row_ptrs_col_inds_transpose.hpp

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include <Kokkos_Core.hpp>
+
+#include "constraints.hpp"
+
+#include "src/restruct_poc/types.hpp"
+
+namespace openturbine {
+
+struct PopulateSparseRowPtrsColInds_Transpose {
+    int rows;
+    int cols;
+    Kokkos::View<int*> col_count;       // cols
+    Kokkos::View<int*> temp_row_ptr;    // cols + 1
+    Kokkos::View<int*> row_ptrs;        // rows + 1
+    Kokkos::View<int*> col_inds;        // nnz
+    Kokkos::View<int*> row_ptrs_trans;  // cols + 1
+    Kokkos::View<int*> col_inds_trans;  // nnz
```

Comment:
make row_ptrs and col_inds `Kokkos::View<int*>::const_type` and put them first (so that the mutable variables come last)